manifests/node.pp

```diff
@@ -54,6 +54,26 @@
 
   Optional[K8s::Firewall] $firewall_type = $k8s::firewall_type,
 ) {
+  include k8s::common
+  include k8s::install::cni_plugins
+
+  if $k8s::manage_container_manager {
+    include k8s::install::container_runtime
+  }
+  if $k8s::manage_repo {
+    include k8s::repo
+  }
+  if $k8s::manage_packages {
+    # Ensure conntrack is installed to properly handle networking cleanup
+    if fact('os.family') == 'Debian' {
+      $_conntrack = 'conntrack'
+    } else {
+      $_conntrack = 'conntrack-tools'
+    }
```

Comment:
```suggestion
    $_conntrack = if fact('os.family') == 'Debian' {
      'conntrack'
    } else {
      'conntrack-tools'
    }
```

Reply:
Went with a selector expression instead, don't like assigning on if-statements. Also added a test on the case. (And also noticed that the Debian OS facts don't include actual OS information in them)

Reply:
oh which data is missing? Do we have a bug in facterDB?

Reply:
The `os` structured fact only included `architecture` on Debian 12 at least.